pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -207,7 +207,8 @@ func newServer(conf *Conf) (*GadgetTracerManager, error) {
 	containerEventFuncs := []pubsub.FuncNotify{}
 
 	if !conf.TestOnly {
-		if _, err := ebpf.RemoveMemlockRlimit(); err != nil {
+		noLimit := unix.Rlimit{Cur: unix.RLIM_INFINITY, Max: unix.RLIM_INFINITY}
+		if err := unix.Prlimit(0, unix.RLIMIT_MEMLOCK, &noLimit, nil); err != nil {
 			return nil, err
 		}
```

Comment:
I think we should use [`RemoveMemlock()`](https://github.com/cilium/ebpf/blob/v0.8.1/rlimit/rlimit.go#L104), it has some additional logic to be no-op when no needed.

Reply:
Good catch, I did not see this function existed hence going `unix`.